examples/kubernetes/requirements.txt

```diff
@@ -0,0 +1,3 @@
+huggingface_hub==0.23.0
```

Comment:
@dmsuehir we have pinned a version < 0.23.0 for `huggingface_hub`. Is there a reason you need this specific version?

Reply:
Yes, [`HF_TOKEN_PATH`](https://huggingface.co/docs/huggingface_hub/en/package_reference/environment_variables#hftokenpath) was introduced in 0.23.0. The alternative is using `HF_HOME`, but that ended up being problematic because mounting the token to that location made `HF_HOME` read only. That caused an issue when other things tried to write to `HF_HOME` (like model/dataset files).